client/a8c-for-agencies/sections/overview/body/next-steps/index.tsx

```diff
@@ -1,5 +1,102 @@
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getAllRemotePreferences } from 'calypso/state/preferences/selectors';
 
-export default OverviewBodyNextSteps;
+import './style.scss';
+
+export default function OverviewBodyNextSteps() {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const preferences = useSelector( getAllRemotePreferences );
+
+	const checkTourCompletion = ( prefSlug: string ): boolean => {
+		if ( preferences && A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ prefSlug ] ) {
+			return A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ prefSlug ] in preferences;
+		}
+		return false;
+	};
+
+	const tasks: Task[] = [
+		{
+			calypso_path: '/sites?tour=sites-walkthrough',
+			completed: checkTourCompletion( 'sitesWalkthrough' ),
+			disabled: false,
+			actionDispatch: () => {
+				dispatch( recordTracksEvent( 'calypso_a4a_overview_next_steps_get_familiar_click' ) );
+				dispatch(
+					savePreference( A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ 'sitesWalkthrough' ], true )
+				);
+			},
+			id: 'get_familiar',
+			title: translate( 'Get familiar with the sites management dashboard' ),
+			useCalypsoPath: true,
+		},
+		{
+			calypso_path: '/marketplace',
+			completed: checkTourCompletion( 'exploreMarketplace' ),
+			disabled: false,
+			actionDispatch: () => {
+				dispatch(
+					recordTracksEvent( 'calypso_a4a_overview_next_steps_explore_marketplace_click' )
+				);
+				dispatch(
+					savePreference( A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ 'exploreMarketplace' ], true )
+				);
+			},
+			id: 'explore_marketplace',
+			title: translate( 'Explore the marketplace' ),
+			useCalypsoPath: true,
+		},
+		{
+			calypso_path: '/sites?tour=add-new-site',
+			completed: checkTourCompletion( 'addSiteStep1' ),
+			disabled: false,
+			actionDispatch: () => {
+				dispatch( recordTracksEvent( 'calypso_a4a_overview_next_steps_add_sites_click' ) );
+				dispatch( savePreference( A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ 'addSiteStep1' ], true ) );
+			},
+			id: 'add_sites',
+			title: translate( 'Learn how to add new sites' ),
+			useCalypsoPath: true,
+		},
+	];
+
+	const numberOfTasks = tasks.length;
+	const completedTasks = tasks.filter( ( task ) => task.completed ).length;
+
+	const isCompleted = completedTasks === numberOfTasks;
+
+	return (
+		<Card>
+			<div className="next-steps">
+				<div className="next-steps__header">
+					<h2>{ isCompleted ? translate( '🎉 Congratulations!' ) : translate( 'Next Steps' ) }</h2>
+					<CircularProgressBar
+						size={ 32 }
+						enableDesktopScaling
+						numberOfSteps={ numberOfTasks }
+						currentStep={ completedTasks }
+					/>
+				</div>
+				{ isCompleted && (
+					<p>
+						{ translate(
+							"Right now there's nothing left for you to do. We'll let you know when anything needs your attention."
+						) }{ ' ' }
```

Comment:
I think we can remove the extra `{ '  ' }` in the line.

Reply:
https://github.com/Automattic/wp-calypso/pull/88986/commits/95b6b585869addc5e6ede4d4ad46f3052f808c28 :+1: